### ISSUE TITLE
About tsm101_tpn pretrained model on sth-sth-v2 dataset

### ISSUE THREAD

**Issue by JR-Wang** [decisionforce/TPN#19]
Hello, may I ask you for your tsm101_tpn pretrained model on sth-sth-v2 dataset?
By the way, I am now change config_files/sthv2/tsm_tpn.py as follow to train a tsm101_tpn model on sth-sth-v2-dataset. 
model = dict(
    type='TSN2D',
    backbone=dict(
        type='ResNet',
        pretrained='modelzoo://resnet101',
        depth=101,
        nsegments=8,
        out_indices=(2, 3),
        tsm=True,
        bn_eval=False,
        partial_bn=False),
As I use 2 gpus, I set my learning rate as 0.0025. However, the top1 precision remains around 56 after 45 epochs training, shall I decay my learning rate?
Looking forward to your reply and many thanks.

**Comment by limbo0000** [comment]
Hi @JR-Wang,

Yes. you can follow the same learning rate scheduler in our config, which decays lr at 75, 125 epoch respectively.
